mystyle.css

```diff
@@ -1,9 +1,92 @@
+p{
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+h1{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px;
+    font-weight: 400;
+}
+
+input{
+    flex: auto;
+}
+
 .topnav{
     display: flex;
+    flex: auto;
+    justify-content: space-between;
 }
 .left-items{
-    
+    justify-content: flex-start;
+}
+
+.central-items {
+    display: flex;
+    justify-content: center;
+    flex:auto;
+}
+#search-input{
+    flex:auto;
+}
+.right-items {
+    justify-content: flex-end;
+}
+
+aside {
+    width:30%;
+    padding-left: 10px;
+    float: right;
+}
+#thumbox{
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    margin: 0%;
+}
+#thumbox-description{
+    margin: 0%;
+    font-family: Arial, Helvetica, sans-serif;
+}
+#video{
+    display: inline-block;
+    width: fit-content;
+    height: fit-content;
+}
+
+.video-title{
+    font-family: Arial, Helvetica, sans-serif;
+}
+#video-uploader{
+    display: flex;
+    justify-content: flex-start;
+}
+.comment-filter{
+    display: flex;
```

Comment:
You could write a class .display-flex (or something like that) and use it where you want it. (suggestion)
I wrote an [article](https://medium.com/free-code-camp/you-just-need-to-know-these-properties-to-start-loving-css-a06aca6087e9) about how to easily use flex if you want to watch it.